frontend/components/GridWrapper/styles.js

```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+import {breakpoints} from '../../styles/breakpoints';
+import {grid} from '../../styles/grid';
+
+export const StyledGridWrapper = styled.section`
+  min-height: calc(100vh - 96px);
+  margin: 48px;
```

Comment:
why not use scale?

Reply:
I can't get scale to work in some places ... might need to book some time with you